scripts/testing/debug_rate_limiter.py

```
@@ -4,14 +4,11 @@
-# pylint: disable=protected-access
-
-import sys
-import os
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
-
-from src.api_rate_limiter import TokenBucketRateLimiter, RateLimitConfig  # noqa: E402
+from pathlib import Path
+from scripts.bootstrap import add_repo_src_to_path, find_repo_root
+repo_root = find_repo_root(Path(__file__))
+add_repo_src_to_path(Path(__file__))
+from src.api_rate_limiter import TokenBucketRateLimiter, RateLimitConfig
```

Ruff (0.12.2)

Module level import not at top of file

(E402)